README.md

```diff
@@ -50,3 +50,10 @@ tests in different environments, where some of them do not have git available.
 When running `catch-uncommitted --skip-node-versionbot-changes`, the script will
 skip checking the `package.json` & the `CHANGELOG.md` for changes, so that it
 can work as part of the balenaCI pipeline.
+
+### --excludes
```

Comment:
NIT can we rename this to `exclude`? I feel like flags should be commands rather than descriptions. i.e. `exclude this` instead of `this excludes that`

Reply:
I also agree with Lucian on this NIT (using a verb).

Reply:
In my mind, it was a noun, as in `a list of things to exclude`. But, yeah, I don't really mind to take off the `s` 😆